addons/batikvis/src/main/java/de/lmu/ifi/dbs/elki/visualization/gui/detail/DetailView.java

```diff
@@ -407,9 +407,7 @@ protected void redraw() {
         vis.incrementalRedraw();
       }
     }
-    if(active || true) {
-      refresh();
-    }
+    refresh();
```

Comment:
I'd prefer to keep this as is, and mark it with a FIXME - clearly, the `refresh()` was intended to be called only if `active` was set, but was force-enabled because of some synchronization issue probably - that may no longer persist.